packages/url_launcher/url_launcher/README.md

```diff
@@ -53,6 +53,28 @@ Common schemes supported by both iOS and Android:
 
 More details can be found here for [iOS](https://developer.apple.com/library/content/featuredarticles/iPhoneURLScheme_Reference/Introduction/Introduction.html) and [Android](https://developer.android.com/guide/components/intents-common.html)
 
+### Encoding URLs
+
+URLs should be safely encoded, espeically when including spaces or other special characters. We can do this using dart's `Uri` helper methods included in `dart:core`:
```

Comment:
Nit: I would suggest "must be properly encoded" rather than "should be safely encoded" (it's not really optional, and "safely" is somewhat ambiguous).